src/app/series/directives/series-feed.component.css

```diff
@@ -35,10 +35,11 @@ li p {
 .futurePublic {
   color: #ffa500;
 }
-
-
 .extra {
   text-align: center;
   margin: 20px;
   font-weight: 400;
 }
+.invalid {
+  color: #e32;
```

Comment:
Are we using this red color anywhere else? Our non-emphasis red is #ee4d3e? Does that not look good?

Reply:
😎 Cool, if we are using it for text elsewhere, let's keep as is! (Invalid color/style probably belongs in the styleguide, but that's something we can evaluate another day with this ticket https://github.com/PRX/styleguide.prx.org/issues/30.)